pkg/crc/preflight/preflight_windows.go

```diff
@@ -141,6 +145,67 @@ var vsockChecks = []Check{
 	},
 }
 
+var adminHelperChecks = []Check{
+	{
+		configKeySuffix:  "check-crc-users-group-exists",
+		checkDescription: "Checking if crc-users group exists",
+		check: func() error {
+			_, _, err := powershell.Execute("Get-LocalGroup -Name crc-users")
+			return err
+		},
+		fixDescription: "Creating crc-users group",
+		fix: func() error {
+			_, _, err := powershell.ExecuteAsAdmin("create crc-users group", "New-LocalGroup -Name crc-users")
+			return err
+		},
+		labels: labels{Os: Windows},
+	},
+	{
+		configKeySuffix:  "check-user-in-crc-users-group",
+		checkDescription: "Checking if current user is in crc-users group",
+		check: func() error {
+			_, _, err := powershell.Execute(fmt.Sprintf("Get-LocalGroupMember -Name crc-users -Member '%s'", os.Getenv("USERNAME")))
+			return err
+		},
+		fixDescription: "Adding current user to crc-users group",
+		fix: func() error {
+			_, _, err := powershell.ExecuteAsAdmin("adding current user to crc-users group", fmt.Sprintf("Add-LocalGroupMember -Group crc-users -Member '%s'", os.Getenv("USERNAME")))
+			if err != nil {
+				return err
+			}
+			return errors.New("Please reboot your system and run 'crc setup' to complete the setup process")
+		},
+		labels: labels{Os: Windows},
```

Comment:
these two checks should also have the `NetworkMode` label?

Reply:
The admin helper is used in both networking mode. So there is no need for that.

Reply:
i found its only used in `user` network-mode, in poststart service we have a check and for `system` network-mode we set the dns address in  vswitch only for user networking mode we use the admin helper.. or do we use it somewhere else aswell?

Reply:
Oh yes good catch! I was sure it was used for both cases, sorry...
I think it's something we forgot to fix. When goodhosts was first introduced, Windows implementation was supposed to be done "later".

Reply:
For instance, https://github.com/code-ready/crc/blob/f113feb287006253db46c647c17ed6ecc89d1d64/pkg/crc/preflight/preflight_checks_common.go#L84 has no label so runs with system network mode.
It uses the admin helper.

Reply:
ahh, yes, thanks!